open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/ControlPanelToolbox.java

```diff
@@ -1,42 +1,62 @@
 package io.opensphere.controlpanels;
 
+import java.awt.Component;
+import java.util.function.Function;
+
+import de.micromata.opengis.kml.v_2_2_0.TimeSpan;
 import io.opensphere.core.PluginToolbox;
-import io.opensphere.core.model.time.TimeSpan;
 import io.opensphere.core.util.ObservableValue;
+import io.opensphere.mantle.data.DataTypeInfo;
 
 /**
  * Defines a toolbox for use with control panels.
  */
 public interface ControlPanelToolbox extends PluginToolbox
 {
+
+    /**
+     * Gets the registry for {@link LayerControlProvider} implementations. This
+     * allows for plugins to register additional providers without injecting
+     * dependencies into the core project.
+     * 
```

Comment:
Remove the extra blank line (you can get Eclipse to remove extra lines for you on save)